tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -3948,4 +3948,172 @@ TestCarePartnerParticipantInfoEditOrAddEmergencyRoomAdmissionsInfo:
       Id: $PAGE.care_platform_edit_participant_hospitalization.delete_emergency_room_admission_dialog_title$
     - Type: wait_for
       Strategy: xpath
-      Id: $PAGE.care_platform_notifications.emergency_room_admission_deleted_notification$
\ No newline at end of file
+      Id: $PAGE.care_platform_notifications.emergency_room_admission_deleted_notification$
+
+#CXTB-111 Participant Profile - Review, Edit or Add Vendor Info
+TestCarePartnerParticipantReviewEditOrAddVendorInfo:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: desktop
+  Actions:
+  # Calling Login case.
+    - Type: case
+      Value: CarePartnerLogin
+  # looking for a senior profile and click on it
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform.navigation_participant$
+    - Type: click 
+      Strategy: xpath
+      Id: $PAGE.care_platform_home.first_call_of_queue$
+  # Click on "records" button 
+    - Type: click
+      Strategy: xpath 
+      Id: $PAGE.care_platform_participant_details.records_button$
+  # Click on "Demographic" and "VENDORS" tab
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_records.demographics_tab$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_records.demographics_vendor_subtab$
+  # Variables to use for Add Vendor
+      Vars:
+        ADDRESS: 1000 Street Name, Somewhere, GA 30024
+        WEBSITE: www.airportridesrus.com
+        CATEGORY: careCoordination
+        PHONE: 4365565897
+    - Type: generate_unique_name
+      Name: 'Airport Rides "R" Us'
+      ResultVar: NAME
+  # Steps to add a vendor
+    # Clicking on "Add Vendor" button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.add_vendor_button$
+    # Waiting for modal
+    - Type: wait_for
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.vendor_modal$
+    # Closing "Add Vendor" modal
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.cancel_vendor_modal_button$
+   # Wait to close modal
+    - Type: wait_not_visible
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.modal$
+    # Open Add "Vendor" modal
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.add_vendor_button$
+  # Validate if "Add vendor" is opened
+    - Type: get_text
+      Strategy: xpath
+      Id : $PAGE.care_platform_edit_participants_vendor.modal_title_label$
+      Greps:
+        - var: MODAL_TITLE
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: eq
+          Var: MODAL_TITLE
+          Value: Add Vendor
+  # Filling Vendor form 
+    - Type: click
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.select_category_modal_input$
+    # Selecting a Category
```

Comment:
Format all comments to have same indentation level

Reply:
I applied the changes in this commit https://github.com/neveralone-chs-org/TestRay/commit/48ed04279808ef48bc2b94fa082f7029b67617a3